src/components/editor/ResourceTemplateForm.jsx

```diff
@@ -62,35 +63,57 @@ class ResourceTemplateForm extends Component {
             <p>BEGIN ResourceTemplateForm</p>
               <div>
                 {this.props.propertyTemplates.map( (pt, index) => {
-                  if(pt.type == 'literal'){
-                    return(
-                      <InputLiteral propertyTemplate = {pt} key = {index} id = {index} />
+
+                  let isResourceWithValueTemplateRefs = Boolean(
+                    pt.type == 'resource' &&
+                    pt.valueConstraint != null &&
+                    pt.valueConstraint.valueTemplateRefs != null
+                    && pt.valueConstraint.valueTemplateRefs.length > 0
+                  )
+
+                  let lookupConfigItem, templateUri, listComponent
+                  try {
+                    templateUri = pt.valueConstraint.useValuesFrom[0]
+                    for(var i in lookupConfig){
+                      lookupConfigItem = Object.getOwnPropertyDescriptor(lookupConfig, i)
+                      if(lookupConfigItem.value.uri === templateUri){
+                        listComponent = lookupConfigItem.value.component
+                        break
+                      }
+                    }
+                  } catch (error) {
+                    console.log(`Some properties were not defined in the json file: ${error}`)
```

Comment:
could/should this message:
1.  be console.error
2.  be clearer - which JSON file?
3.  does ${error} display real info or does it display "object"?   should it be `${util.inspect(error)}` ?

Reply:
${error} displays real info, e.g. `TypeError: Cannot read property 'useValuesFrom' of undefined`

If I change it to error, we will get a lot of red lines in otherwise passing tests for templates that do not have `useValuesFrom` as well as in the console. The try/catch here is not totally intended to catch errors, but also to otherwise allow rendering where it would normally break because there was not a value in the template needed for setting the variables. I suppose I could check for the undefined variables instead of using try/catch here.

Reply:
Removed try/catch altogether.